README.md

```diff
@@ -69,6 +82,18 @@ warn_on_mons_with_no_ac = 2
 #
 # Set to 0 to disable.
 bluetooth_low_pct = 40
+
+# Set to false to disable state change notifications e.g. when charging, discharging, reaching the threshold
+state_notif_enabled = true
+
+# Positive values: Expiry time for the respective notification in milliseconds. 
+# 0:  Do not expire, user will have to close the notification manually.
+# Negative values: Expire according to server default.
+sleep_pct_notif_timeout = 0
+low_pct_notif_timeout = 0
+state_notif_timeout = -1
```

Comment:
Here and elsewhere, instead of using values like 0 (disable), -1 (default behavior), or 0 (no timeout), serialise as tagged unions to make the intention explicit.

Reply:
(`warn_on_mons_with_no_ac` is already violating this, but let's not add more)

Reply:
I guess that is better UX wise, this was just how the underlying notification library did it. Still the user will need to consult the documentation anyways to know the exact keywords, no?

Reply:
Just saw your second comment just now (after refreshing the page). So basically 0 (never), negative (server default), and positive (no timeout) is only for the notification timeouts.